onnxruntime/core/session/onnxruntime_c_api.cc

```diff
@@ -2762,8 +2763,12 @@ ORT_API(const OrtApi*, OrtApis::GetApi, uint32_t version) {
   return nullptr;  // Unsupported version
 }
 
-ORT_API(const char*, OrtApis::GetVersionString) {
-  return ORT_VERSION;
+ORT_API(const ORTCHAR_T*, OrtApis::GetVersionString) {
+  return ORT_TSTR(ORT_VERSION);
```

Comment:
I probably gave you a bad suggestion. It did not work.

Reply:
@snnn Yes, it looks like that `ORT_TSTR` should not be applied to a macro, which may lead to undefined variables. I define another macro `ORT_TSTR_ON_MACRO` to do so. I create a code snippet to replay the compile error in Linux and found the new macro could solve the problem. I hope it will also work in Windows which I don't have.
![image](https://user-images.githubusercontent.com/19584326/235053317-fcdad8d3-f26e-47bd-b6d3-a03f22572fdc.png)